### ISSUE TITLE
离线使用docker-compose部署的harbor缺少开机自启动，希望可以完善

### ISSUE THREAD

**Issue by xlogin** [kubesphere/kubekey#1303]
### What is version of KubeKey has the issue?

2.1.0

### What is your os environment?

centos 7.9

### KubeKey config file

```yaml
离线使用docker-compose部署的harbor缺少开机自启动，希望可以完善
```

### A clear and concise description of what happend.

离线使用docker-compose部署的harbor缺少开机自启动，希望可以完善

### Relevant log output

```shell
离线使用docker-compose部署的harbor缺少开机自启动，希望可以完善
```

### Additional information

离线使用docker-compose部署的harbor缺少开机自启动，希望可以完善

**Comment by 24sama** [comment]
Hi @xlogin 
I think this PR #1302 which is contributed by @Feeeenng can solve this problem. Welcome to review it together!

**Comment by 24sama** [comment]
Fixed in [v2.1.1](https://github.com/kubesphere/kubekey/releases/tag/v2.1.1)
